systemtest/src/main/java/io/strimzi/systemtest/resources/crd/KafkaBridgeResource.java

```diff
@@ -89,9 +90,12 @@ private static KafkaBridge getKafkaBridgeFromYaml(String yamlPath) {
     }
 
     private static KafkaBridge waitFor(KafkaBridge kafkaBridge) {
-        LOGGER.info("Waiting for Kafka Bridge {}", kafkaBridge.getMetadata().getName());
-        DeploymentUtils.waitForDeploymentReady(kafkaBridge.getMetadata().getName() + "-bridge", kafkaBridge.getSpec().getReplicas());
-        LOGGER.info("Kafka Bridge {} is ready", kafkaBridge.getMetadata().getName());
+        String name = kafkaBridge.getMetadata().getName();
```

Comment:
```suggestion
        String kafkaBridgeCrName = kafkaBridge.getMetadata().getName();
```
and same applied with the others...